.github/workflows/docs.yml

```diff
@@ -0,0 +1,43 @@
+# Simple workflow for building documentation
+name: Build documentation
+
+# On a pull request this will build the html documentation
+# and create an artifact. It does this by using the standard
```

Comment:
nit: "standard" build target made it sound to me like a github thing. I think we could be clearer here.